sdcm/nemesis.py

```diff
@@ -675,14 +675,15 @@ def _kill_scylla_daemon(self):
 
     @target_all_nodes
     def disrupt_stop_wait_start_scylla_server(self, sleep_time=300):  # pylint: disable=invalid-name
```

Comment:
I would recommend using @decorate_with_context, it would be less of interruption to the code